<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CastingWidget</class>
 <widget class="QWidget" name="CastingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <widget class="QGroupBox" name="issueGroupBox">
     <property name="title">
      <string>Receive Metal and Diamond</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ranar Wt.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Product Wt.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Dia Wt.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Dia Pcs.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="ranarWtDoubleSpinBox"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="productWtDoubleSpinBox"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="diaWtDoubleSpinBox"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QSpinBox" name="diaPcsSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="receiveGroupBox">
     <property name="title">
      <string>Detail</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="pcsSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pcs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="issueDateEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Issue Date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="jobNoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="detailGroupBox">
     <property name="title">
      <string>Issue Metal and Diamond</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Issue Weight</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Metal Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Purity</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Issue Dia Wt.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Issue Dia Pcs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="metalPurityComboBox"/>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="metalNameComboBox"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QSpinBox" name="issueDiaPcsSpinBox"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="issueDiaWtDoubleSpinBox"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="issueMetalWtDoubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
